apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -255,19 +319,64 @@ def test_notes_query_filter(
             }
         """
 
-        filters: dict[str, Any] = {}
+        with self.assertNumQueriesWithoutCache(expected_query_count):
+            response = self.execute_graphql(query, variables={"filters": filters})
 
-        if case_manager_label:
-            filters["createdBy"] = getattr(self, case_manager_label).pk
+        if teams == ["invalid team"]:
+            self.assertIsNone(response["data"])
+            self.assertEqual(len(response["errors"]), 1)
+            self.assertIn("does not exist in 'SelahTeamEnum'", response["errors"][0]["message"])
 
-        if client_label:
-            filters["client"] = getattr(self, client_label).pk
+        else:
+            self.assertEqual(response["data"]["notes"]["totalCount"], expected_results_count)
+            notes = response["data"]["notes"]["results"]
 
-        if search_terms:
-            filters["search"] = search_terms
+            for idx, note_label in enumerate(returned_note_labels):
+                self.assertEqual(notes[idx]["id"], getattr(self, note_label)["id"])
 
-        if is_submitted is not None:
-            filters["isSubmitted"] = is_submitted
+    @parametrize(
+        ("search_terms, expected_results_count, returned_note_labels"),
+        [
+            ("deets", 2, ["note_2", "note_3"]),
+            ("deets coop", 1, ["note_2"]),
+            ("more", 1, ["note_3"]),
```

Comment:
maybe make these more realistic examples to increase readability?

Reply:
I added testcase comments